src/containers/Home.js

```diff
@@ -1,38 +1,56 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
+import Pagination from "material-ui-flat-pagination";
 
-import { getMovies } from '../store/actions/MovieActions';
+import { getMovies, getMoviesByPage, getMoviesCount } from '../store/actions/MovieActions';
 import MovieCard from '../component/MovieCard';
 
 class Home extends Component {
+  
   componentDidMount() {
-    this.props.getMovies();
+    this.props.getMoviesCount();
+    this.props.getMoviesByPage({offset: 0, take:5});
+  }
+  state ={
+    offset : 0,
   }
-
   renderMovies = () => {
     return this.props.movies.map(movie => <MovieCard key={movie.id} movie={movie} />);
   };
```

Comment:
ovo izbjegavaj, napravi `MoviesList` komponentu

Reply:
Imam task na PR-u za stilizovanje MoviesList pa cu to na toj grani da odradim

Reply:
vazi, samo da znas da ovaj komentar nema nista sa stilom, ovo je vise nacin pravljenja komponenti, izbjegavaj da pravis `render`Nesto`` funkcije, genralno se za sve moze napraviti nova komponenta